apis/core/v1alpha1/restoresession_types.go

```diff
@@ -127,6 +127,9 @@ type KubeDBManifestOptions struct {
 
 // RestoreDataSource specifies the information about the data that will be restored
 type RestoreDataSource struct {
+	// Namespace specifies the Namespace name where the DataSource i.e (Repository, Snapshots) exists.
```

Comment:
Namespace specifies the namespace of the DataSource (i.e. Repository, Snapshot).